MergerService/Runners/ITaskExecutor.cs

```diff
@@ -0,0 +1,10 @@
+using MergerService.Models.Tasks;
+using MergerService.Utils;
+
+namespace MergerService.Runners
+{
+  public interface ITaskExecutor
+  {
```

Comment:
instead of beeing defined in a separate file you can put it above the class definition in the TaskExecutor file

Reply:
IMO this is fine just as we did for different interfaces in this project (such as ITileMerger).